projects/transactions-journey/src/lib/components/transaction-item/transaction-item.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { TransactionItemComponent } from './transaction-item.component';
+
+describe('TransactionItemComponent', () => {
+  let component: TransactionItemComponent;
+  let fixture: ComponentFixture<TransactionItemComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ TransactionItemComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(TransactionItemComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Would you like to add more unit tests to cover multiple ways to test component? For example, simple unit test, and shallow component testing.

Reply:
I will add unit tests as part of the scope of this ticket if you don't mind https://backbase.atlassian.net/browse/WF-336

Reply:
Sure. 👍